distributed/distributed.yaml

```diff
@@ -30,6 +30,7 @@ distributed:
       rechunk-split: 1us
       shuffle-split: 1us
     validate: False         # Check scheduler state at every step for debugging
+    zeroconf: true
```

Comment:
Call me paranoid but is this a secure default? can this discovery / announcement be harmful? (I have no idea about zeroconf)

Reply:
I had the same thoughts about this. Here are some thoughts that lead me to this decision.

- Zeroconf announces the IP and port of a specific service on a local network. If a Dask cluster is running without TLS that does mean it will be easier for others to find and use it. 
- However running an unsecured cluster on a network can still be discovered and used by other means (port scanning).
- mDNS entries are used on the local LAN only, so it's not like clusters exposed to the internet will announce themselves to the whole internet.
- I've not included the `zeroconf` dependency in the distributed requirements, so it will not be used on a clean install. 
- I do intend to make it a dependency of dask-ctl which will activate it as a side effect of installing that tool.
- The goal here is to make discovery easy. Having it disabled by default kinda defeats that.

So my feeling is that this is not harmful. 

Perhaps an analogy would be that zeroconf is like telling everyone on your street where you live. They could probably figure this out if they wanted to. And the most important thing is locking your door anyway.

Reply:
nice summary, thank you. that's good enough for me